crates/ruff_python_parser/src/parser/statement.rs

```diff
@@ -1,5 +1,7 @@
 use std::fmt::Display;
 
+use itertools::Itertools;
+
```

Comment:
Nit: Uh no :P I don't like itertools haha. It has a few extensions that make it non obvious that they're allocating.

Reply:
I think I only use it for `join` which can be done with fold as well.